packages/forms-web-app/src/dynamic-forms/dynamic-components/boolean/question.js

```diff
@@ -25,7 +25,7 @@ class BooleanQuestion extends RadioQuestion {
 		description,
 		html,
 		validators,
-		type = 'radio',
+		interfaceType = 'radio',
```

Comment:
This frees up the key `type` to describe the type of question. See question-props.d.ts